.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('eslint-doc-generator/dist/lib/options').GenerateOptions} */
```

Comment:
This won't work anymore and needs to be switched to the public type:

```suggestion
/** @type {import('eslint-doc-generator').GenerateOptions} */
```